src/Data/Lens/Setter.purs

```diff
@@ -40,28 +41,28 @@ set l b = over l (const b)
 (.~) = set
 
 (+~) :: forall s t a. (Semiring a) => Setter s t a a -> a -> s -> t
-(+~) p = over p <<< flip (+)
+(+~) p = over p <<< add
```

Comment:
Yeah :smile:, you worried me for a minute there, but that was indeed my thinking.